cellxgene_schema_cli/tests/test_validate.py

```diff
@@ -306,13 +306,15 @@ def test__validate_with_h5ad_invalid_and_without_labels(self):
         assert errors
         assert is_seurat_convertible
 
+
+class TestCheckSpatial:
     def test__validate_spatial_visium_ok(self):
         validator: Validator = Validator()
         validator._set_schema_def()
         validator.adata = adata_visium.copy()
 
         # Confirm spatial is valid.
-        validator._check_spatial()
+        validator._check_spatial_uns()
```

Comment:
Updated existing uns-specific tests to call `_check_spatial_uns`, here and throughout.